### ISSUE TITLE
Introduce support for systemd init system

### ISSUE THREAD

**Issue by fpietrosanti** [globaleaks/GlobaLeaks#1840]
This ticket is to introduce support in GlobaLeaks package for systemd init system.

Systemd, since ubuntu 16.04, do provide a variety of functional features as an init system (despite the philosophical battle around it) including application watchdog support.

This ticket is to analyze technically the implementation of a systemd compatible init system for GlobaLeaks, exploiting at maximum it's features.

**Comment by ralienpp** [comment]
Being familiar with `systemd, I could provide assistance with this. However, I need to know what the expected results are.

- Does Globaleaks consist of multiple processes?
- What are the dependencies?
- In which order do they have to be started?

In other words, in order to implement this logic using `systemd`, the requirements have to be expressed in plain English first.

**Comment by evilaliv3** [comment]
hello @ralienpp ! thanks for supporting this idea,

here the answers to your current question. feel free to reach out to us if you have more.

1) multiple processes but only one father with multiple children, currently a core + some https deamons, but in the future there will be more deamons handling other actions (e.g. mail spooling)
2) the dependencies are written here: https://github.com/globaleaks/GlobaLeaks/blob/master/debian/controlX/control.xenial
3) the process that spown all is just one, "globaleaks" then it internally spawn all the others

**Comment by NSkelsey** [comment]
This ticket is now outdated.

Cursory support for systemd has been introduced via the script `debian/globaleaks.init` loaded like always at  `/etc/init.d/globaleaks`. Since further discussion of advanced features of systemd like application profiles has been only brought up when the sandboxing broke communication with Tor over `/var/globleaks/torhs`.
